frontend/awx/resources/hosts/hooks/useHostsFilters.tsx

```diff
@@ -17,11 +17,24 @@ export function useHostsFilters() {
   const params = useParams<{ inventory_type: string }>();
 
   const toolbarFilters = useMemo<IToolbarFilter[]>(() => {
-    const filters = [nameToolbarFilter, descriptionToolbarFilter, modifiedByToolbarFilter];
+    let filters: IToolbarFilter[] = [];
 
-    if (params.inventory_type === 'inventory' || !params.inventory_type) {
-      filters.push(createdByToolbarFilter);
+    if (!params.inventory_type || params.inventory_type === 'inventory') {
+      filters = [
+        nameToolbarFilter,
+        descriptionToolbarFilter,
+        createdByToolbarFilter,
+        modifiedByToolbarFilter,
+      ];
```

Comment:
```suggestion
    if (!params.inventory_type || params.inventory_type === 'inventory') {
      filters = filters.splice(1, 0, descriptionToolbarFilter)
```

I think that'll work to make this whole conditional a bit shorter.